react/react1/week3/reactweek3/src/DataFetching.js

```diff
@@ -0,0 +1,24 @@
+// import { useEffect } from "react";
+
+// const Api =
+//   "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+//   useEffect(() => {
+//     fetch(Api)
+//       .then((res) => res.json())
+//       .then((data) => {
+//         const fetchMap = data.map((data) => ({
+//           id: data.id,
+//           description: data.description,
+//           deadline: data.deadline,
+//         }));
+
+//         console.log(setUseTodo);
+
+//         setUseTodo((prev) => [...prev, ...fetchMap]);
+//       });
+//   }, [setUseTodo]);
+// export function DataFetching({ setUseTodo }) {
+//   //   const [fetchTodo, setFetchTodo] = useState([]);
+
+//   return null;
+// }
```

Comment:
I would delete this file if it is not in use.

Reply:
done